.github/workflows/build-macos-media-playback.yml

```
@@ -0,0 +1,54 @@
+
+jobs:
+  build-v2-darwin:
+    name: Build v2 on macOS
+    if: contains(github.ref, 'v2')
+    runs-on: macos-latest
+    env:
+      GOWORK: "off"
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)